modules/knative-serving-advanced-config.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * /serverless/installing_serverless/serverless-install-config-options.adoc
+
+[id="knative-serving-advanced-config_{context}"]
+= Knative Serving supported installation configuration options
+
+This guide provides information for cluster administrators about advanced installation configuration options for Knative Serving.
+
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field. Some of the configuration values in this field are injected by the {ServerlessOperatorName}, and modifying them will cause your deployment to become unsupported.
+====
+
+image::serving-form-view.png[Create Knative Serving form in web console Administrator Perspective]
+
+== Controller Custom Certs
+
+If your registry uses a self-signed certificate, you must enable tag-to-digest resolution by creating a ConfigMap or Secret.
+The {ServerlessOperatorName} then automatically configures Knative Serving controller access to the registry.
+
+To enable tag-to-digest resolution, the Knative Serving controller requires access to the container registry.
+
+[IMPORTANT]
+====
+The ConfigMap or Secret must reside in the same namespace as the Knative Serving custom resource definition.
```

Comment:
s/custom resource definition/CustomResourceDefinition/

or since a lot of people know them as CRDs:

s/custom resource definition/CustomResourceDefinition (CRD)/